core/meterer/meterer.go

```diff
@@ -0,0 +1,57 @@
+package meterer
+
+import (
+	"time"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+)
+
+// Config contains network parameters that should be published on-chain. We currently configure these params through disperser env vars.
+type Config struct {
+	// for rate limiting 2^64 ~= 18 exabytes per second; 2^32 ~= 4GB/s
+	// for payments      2^64 ~= 18M Eth;                2^32 ~= 4ETH
+	GlobalBytesPerSecond uint64 // Global rate limit in bytes per second for on-demand payments
+	MinChargeableSize    uint32 // Minimum size of a chargeable unit in bytes, used as a floor for on-demand payments
+	PricePerChargeable   uint32 // Price per chargeable unit in gwei, used for on-demand payments
+	ReservationWindow    uint32 // Duration of all reservations in seconds, used to calculate bin indices
+
+	ChainReadTimeout time.Duration // Timeout for reading payment state from chain
+}
+
+// disperser API server will receive requests from clients. these requests will be with a blobHeader with payments information (CumulativePayments, BinIndex, and Signature)
```

Comment:
nit: `Meterer handles payment accounting across different accounts. Disperser API server receives requests from clients and each request contains a blob header....`